src/Microsoft.DotNet.Interactive.PowerShell/Commands/OutDisplayCommand.cs

```diff
@@ -54,7 +55,21 @@ public sealed class OutDisplayCommand : PSCmdlet
         /// </summary>
         protected override void ProcessRecord()
         {
+            
             object obj = InputObject is PSObject psObject ? psObject.BaseObject : InputObject;
+
+            // If object is PSCustomObject, convert to PSObject to find the properties to display
+            if (InputObject is PSObject && ((PSObject)InputObject).BaseObject is PSCustomObject)
```

Comment:
you should be able to do something like:

```suggestion
            if (InputObject is PSObject psObj && psObj.BaseObject is PSCustomObject)
```

which will create a `psObj` variable that you can use later if `InputObject is PSObject`